cmd/smashwords-downloader/main.go

```diff
@@ -21,16 +31,36 @@ const (
 	bookListSize     int    = 20 // Number of books on each smashwords list page
 )
 
-func createBookFileName(title string) string {
+//parser is a part of the goreader repo for parsing epubs
```

Comment:
nit: There should be a space after the `//`

This happens again on lines 43 and 197
```suggestion
// parser is a part of the goreader repo for parsing epubs
```